.github/workflows/update-webmentions.yml

```diff
@@ -0,0 +1,48 @@
+name: Update Webmentions
+
+on:
+  workflow_dispatch: # Allow running on-demand
+  schedule:
+    # Every Monday at 8:00 UTC (4:00 Eastern)
+    - cron: '0 8 * * 1'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  update:
+    name: Update Webmentions
+    runs-on: ubuntu-latest
+    env:
+      BRANCH_NAME: auto-webmentions
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: .nvmrc
+          cache: yarn
+      - run: yarn install
+      - run: yarn eleventy
```

Comment:
In reality we don't need a full re-build, just a way to run `webmentions.cjs` to update `webmentions.json`. I couldn't figure out a way to do it though 😅

Reply:
Updated to run this as a standalone `yarn webmentions` command instead of automatically during the build.